### ISSUE TITLE
zabbix_proxy support

### ISSUE THREAD

**Issue by rmdevheart** [patricegautier/unifiZabbix#69]
So all this looks fine, but what i need to do if my unifi setup monitored via zabbix-proxy and zabbix-server couldn't reach devices via ssh directly.

**Comment by lslamp** [comment]
@rmdevheart 
I also had major issues getting ssh to work with my unifi monitoring.
I have followed two different routes.
1. Installed unifi-poller , grafana with influxdb, got that all working very well. From that I got 6 dashboards, using on grafana.
2. I installed each device into zabbix as a unit, then added geeric linux SNMP templates. The worked for my USG, switches and APs

If you want the exact details reach out to me on my email address. lslamp@hotmail.com

Lawrence
